staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -108,3 +118,12 @@ if [ "${GENS}" = "all" ] || grep -qw "informer" <<<"${GENS}"; then
            --output-package "${OUTPUT_PKG}/informers" \
            "$@"
 fi
+
+if [ "${GENS}" = "all" ] || grep -qw "openapi" <<<"${GENS}"; then
+  echo "Generating OpenAPI definitions for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/openapi"
+  ${GOPATH}/bin/openapi-gen \
+           --input-dirs $(codegen::join , "${EXT_FQ_APIS[@]}") \
+           --input-dirs "k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/runtime,k8s.io/apimachinery/pkg/version" \
```

Comment:
we need `k8s.io/api/autoscaling/v1` to generate `v1.Scale` definition for publishing crd spec

Reply:
fixed